Controllers/Controller.cs

```diff
@@ -0,0 +1,60 @@
+using Microsoft.AspNetCore.Mvc;
+using todoDotNet6.TodoRepo;
+
+namespace todoDotNet6.Controllers
+{
+    [ApiController] 
+    [Route("task")]
+    public class Controller : ControllerBase
+    {
+        private readonly ITodoRepository todoRepository;
+
+        public Controller(ITodoRepository todoRepository)
+        {
+            this.todoRepository = todoRepository;
```

Comment:
`this.` isn't necessary if you follow the `_` convention we talked about (please do)